### ISSUE TITLE
An option to zoom in/out just by using the mousewheel (without Ctrl pressed)

### ISSUE THREAD

**Issue by alexandervrs** [zrispo/wick#638]
Could we have an option to zoom in/out just by using the mousewheel (without having Ctrl pressed)?
As I use a lot of art software like SAI or Clip Studio that do this, it's faster this way when I am also using a pen tablet.

**Comment by zrispo** [comment]
Yup, that'd be an easy thing to add in. I'll try to get to it this weekend~

**Comment by alexandervrs** [comment]
Thanks :+1:
